eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -42,6 +43,7 @@ steps:
         ${{ parameters.SubscriptionConfiguration }}
       "@ | ConvertFrom-Json -AsHashtable;
 
+<<<<<<< HEAD
```

Comment:
looks like a unfinished merge.

Reply:
Fixed. Sorry about that. Testing the changes in other repos and didn't inspect my own copy 👎